eodatasets3/assemble.py

```diff
@@ -219,6 +220,10 @@ def __init__(
             self.names = ComplicatedNamingConventionsDerivatives.for_c3_derivatives(
                 self
             )
+        elif naming_conventions == "deafrica_usgs_c2":
```

Comment:
Should this just be a "deafrica" naming convention? It's not currently defining anything Landsat specific.

... otherwise we're going to be adding a separate one for every single deafrica product in the future, right?

Reply:
We might need one for deafrica_s2...

Reply:
I think this will work for both WO and FC. I guess I'm happy to just call it `deafrica` for now, though. I'll change it.